### PR TITLE
remove "TODO clean after pjsip migration"

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: xivo_dao/asterisk_conf_dao.py [hunk 1]
```diff
@@ -918,14 +918,8 @@ def is_user_in_group(row):
                 state_interface='hint:{}@usersharedlines'.format(row.uuid),
             )
         else:
-            # TODO clean after pjsip migration
```
[the remark this answers is not in the document]
Comment (fblackburn1) [reply xivo_dao/asterisk_conf_dao.py:921]: Fixed, thank you
DB fix: https://github.com/wazo-platform/xivo-manage-db/pull/130

**Comment by fblackburn1** [comment]
recheck

**Review by pc-m — APPROVED** [review]

**Comment by fblackburn1** [comment]
recheck

**Comment by rgarrigue** [comment]
recheck

**Pull request merged**